blocks/HARDWARE/PROTOCOLS/CAN/BUS/RECEIVE_CAN_MESSAGE/RECEIVE_CAN_MESSAGE.py

```diff
@@ -0,0 +1,44 @@
+from flojoy import flojoy, DataContainer, DeviceConnectionManager, Stateful
+from typing import Optional
+import can
+
+
+@flojoy(deps={"python-can": "4.3.1"})
+def RECEIVE_CAN_MESSAGE(
+    CAN_address: str,
```

Comment:
We should probably include these in the front end at some point

Reply:
I agree, but there are various CAN interfaces, some requiring specific strings like for PEAK to instanciate the connection with the driver, and others just requiring the serial address like CANABLE. Since we don't provide connections for all possible interfaces, it could cause extensibility problem if someone needs an interface that is not currently supported and try to add the CONNECT block to its interface and try to use the other CAN blocks.